scripts/pre_commit_linter.py

```diff
@@ -535,6 +535,12 @@
     '/scripts/install_third_party_libs.py',
     '/.github/']
 
+# Check if project is running in CircleCI.
+try:
+    CIRCLECI_ENV = os.environ['CIRCLECI']
+except KeyError:
+    CIRCLECI_ENV = False
```

Comment:
Instead of this, use https://docs.python.org/2.7/library/os.html#os.getenv instead.

Reply:
Done.